l5kit/l5kit/data/zarr_dataset.py

```diff
@@ -61,6 +62,9 @@ def __init__(self, path: str, key: str = ""):
         """
         self.key = key
         self.path = path
+        self.frames = np.empty(0, dtype=FRAME_DTYPE)
+        self.scenes = np.empty(0, dtype=SCENE_DTYPE)
+        self.agents = np.empty(0, AGENT_DTYPE)
```

Comment:
Minor: `self.agents = np.empty(0, dtype=AGENT_DTYPE)`

Reply:
I am not a fan of having three function to initialize dataset XD.
- `__init__`
- `initialize`
- `open`